### ISSUE TITLE
dynamic app.R drop down menu

### ISSUE THREAD

**Issue by rbutleriii** [Longo-Lab/de_dashboards#9]
Related to #8 , it would be good if the `app.R` script had a dynamic drop down menu that only showed the valid typeouts that won't generate an error. This one will be more challenging.

**Comment by cyouh95** [comment]
Looks like the current error in the `PS19_C31_snRNAseq` dashboard is due to wrong `typeouts` specified in its `deploy.R`, causing the needed files not to be pushed. Specifically, ` ` & `/` were not properly replaced by `_` & `-`. This should now be fixed by #8.

If we further want to make sure the drop down menu only shows cell types that have saved dashboard files, maybe we can filter `projs` in `app.R`:

```r
typeouts <- list.files(pattern = '\\.dashboard_files\\.rdata', recursive = T) %>%
  str_extract('[^/]+')

lapply(projs, function(x) x[x %in% typeouts])
```

**Comment by rbutleriii** [comment]
Yeah, reviewing the process to deploy, the choice of subclasses to run for the differential expression script is hard coded into `00-commands.sh`, so there is probably not an easy way to automate which subclasses will be pulled in. Going to close this one as a `wontfix`. We can come back to it at some point, but too few resources.
